install_config/cluster_metrics.adoc

```diff
@@ -969,7 +969,7 @@ additional rules variable:
 openshift_prometheus_additional_rules_file: <PATH>
 ----
 
-The file content should be in Prometheus Alert rules format. The following
+The file content should be link:https://prometheus.io/docs/prometheus/latest/configuration/alerting_rules/[in Prometheus Alert rules format]. The following
```

Comment:
To ditch the modal, I'd say "This file must follow the xref:[Prometheus Alert rules format]."